x11-libs/libdlo/metadata.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>ck+gentoo@bl4ckb0x.de</email>
+		<name>Conrad Kostecki</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription>
+		A fully open source (GPL) driver which supports all DisplayLink devices
+		to date (all DL1xx devices) is in the mainline of the Linux 2.6.38 kernel,
+		and can be compiled as a module for certain older kernels. It is called udlfb.
+		It's still very much under development, but it's possible to write custom applications
+		to it today (or to libdlo user mode library itself, bypassing udlfb).
+		And it's also possible to configure X and standard X applications to run on udlfb
+		but that part is still very distro dependent today. Most people in the DisplayLink
```

Comment:
That description seems to be overly long, even if it is called `longdescription`. At least the last two sentences seem superfluous. And what about the kernel number reference relative to 'to date' - it could be a distraction or make it appear this being a very old (?) package contrary to being 'still very much under development'.

Reply:
I've changed the description, which shoud now sound better.